iwconfig.js

```diff
@@ -95,6 +95,10 @@ function parse_status_block(block) {
     parsed.unassociated = true;
   }
 
+  if ((match = block.match(/Bit Rate[:|=]\s*([0-9\.]+)/))) {
```

Comment:
Redundant parentheses but better you keep to the existing style I guess

Reply:
this is a C/C++ style/warning/lint thing, clarifies you didn't just accidentally use = instead of ==

Reply:
Didn't notice the single `=`... :pepebarf: